app/views/defence_requests/show.html.erb

```diff
@@ -31,7 +31,7 @@
       <%= render "defence_requests/show_partials/interview_tab" %>
     </div>
     <div id="case-details">
-      <h3 class="nonjs"><%= t("detention_details")%></h3>
+      <h3 class="nonjs"><%= t("case_details")%></h3>
```

Comment:
So is `Case Details` the final copy? Because the latest design was using `Detention Details` and I've put it in a few places in the code.